exercises/concept/valentines-day/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Learn about algebraic data types by deciding what activity to surprise your partner with on Valentines Day.",
+  "icon": "valentines-day",
```

Comment:
There is no need to specify the icon if its slug matches that of the exercise (which it does).